src/views/ForgotPasswordPageChange/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import {Link, Route} from 'react-router-dom';
+import LoginPage from '../LoginPage/index';
+
+var requirements = {
+  password: undefined,
+  confirmpassword: undefined,
+}
+
+class ForgotPasswordPageChange extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      code: undefined
+    }
+  }
+
+  handleChange = event => {
+
+    const info = event.target.name;
+    const value = event.target.value;
+
+    this.setState({
+        [info]: value
+      }
+    );
+    console.log(this.state);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    requirements.password = this.state.code;
+    requirements.confirmpassword = this.state.confirmpassword;
+    console.log(this.state);
```

Comment:
We'll want to get rid of console.logs in future, but for the purpose of testing/debugging, it's ok for now!